rero_ils/query.py

```diff
@@ -189,6 +189,9 @@ def items_search_factory(self, search, query_parser=None):
     """Search factory for item records."""
     search, urlkwargs = search_factory(self, search)
     view = request.args.get('view')
+    organisationPid = request.args.get('organisation')
```

Comment:
Use only small case letters for variable PEP8 recommendations). 
e.g. : 'org_pid' or `organisation_pid`